plugins/turing/go.mod

```diff
@@ -19,6 +19,8 @@ require (
 	golang.org/x/oauth2 v0.0.0-20221014153046-6fdb5e3db783
 )
 
+require github.com/google/uuid v1.3.0
```

Comment:
Is this intentional? - Why is this dependency not in the above `require` block?

Reply:
Hmm good point; not intentional but autogenerated from `go mod tidy`. Lemme poke this around 🤔

Reply:
Okay it seems like after manually shifting it and then running `go mod tidy`, the dependency stays in the block above now 👀